src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -57,7 +57,8 @@ public static IEnumerable<RegexPattern> HighConfidenceSecurityModelsIterator()
         new Unclassified32ByteBase64String(),
         new Unclassified64ByteBase64String(),
         new AadClientAppLegacyCredentials34(),      // SEC101/101
-        new Unclassified16ByteHexadecimalString(),         
+        new Pkcs12CertificatePrivateKeyBundle(),
+        new Unclassified16ByteHexadecimalString(),
```

Comment:
>Unclassified16ByteHexadecimalString [](http://example.com/codeflow?start=12&length=35)

hey! just saying, this value is probably a GUID in many contexts. :) one that's been formatted to elide the hyphens, allowing for the same uniqueness in a slightly smaller payload